tests/unit/actions/indexing/indexable-post-indexation-action-test.php

```diff
@@ -156,7 +206,12 @@ public function test_index() {
 		Filters\expectApplied( 'wpseo_post_indexation_limit' )->andReturn( 25 );
 
 		$this->post_type_helper->expects( 'get_public_post_types' )->once()->andReturn( [ 'public_post_type' ] );
-		$this->wpdb->expects( 'prepare' )->once()->with( $expected_query, [ 'public_post_type', 25 ] )->andReturn( 'query' );
+		$this->post_type_helper->expects( 'get_excluded_post_types_for_indexables' )->once()->andReturn( [] );
+
+		$this->wpdb->expects( 'prepare' )->once()->with(
```

Comment:
Can you add each method call to a new line, because this line is a little bit messy.